webpack/utils/helpers.js

```diff
@@ -81,6 +81,7 @@ export const sendErrorNotifications = messages => (dispatch) => {
     const message = typeof msg === 'string' ? msg : `${msg.message}: ${msg.details}`;
     dispatch(addToast({
       type: 'error',
+      key: 'toastError',
```

Comment:
Could there be more than on error thrown at the same time? not sure we should use a fixed key here.

Reply:
Well, I'm not sure either. But I'd find a bad UX in multiple toasts thrown at me with error. On the other hand, it's possible that different components would call different APIs and each of them fails with a toast... :/

Reply:
Indeed, they might even have different error messages

Reply:
this seems like a good spot to use the forEach((msg, index) => 
... 
key: `toastError${index}`,
... 
and disable the eslint rule around such things for this line. no?